crates/matrix-sdk-base/src/deserialized_responses.rs

```diff
@@ -310,3 +464,104 @@ impl SyncOrStrippedState<RoomPowerLevelsEventContent> {
         }
     }
 }
+
+#[cfg(test)]
+mod test {
+    macro_rules! assert_display_name_eq {
+        ($left:expr, $right:expr $(, $desc:expr)?) => {{
+            let left = crate::deserialized_responses::DisplayName::new($left);
+            let right = crate::deserialized_responses::DisplayName::new($right);
+
+            similar_asserts::assert_eq!(
+                left,
+                right
+                $(, $desc)?
+            );
+        }};
+    }
+
+    macro_rules! assert_display_name_ne {
+        ($left:expr, $right:expr $(, $desc:expr)?) => {{
+            let left = crate::deserialized_responses::DisplayName::new($left);
+            let right = crate::deserialized_responses::DisplayName::new($right);
+
+            assert_ne!(
```

Comment:
Time to contribute to `similar_asserts` to include `assert_ne!`?

Reply:
What would you then show? The `ne` variant fails if the two given objects are equal but they shouldn't be. Two equal objects create an empty diff, so what should `similar_asserts` show?

Reply:
🤦 

/me goes back in its bed